CONTRIBUTING.md

```diff
@@ -110,6 +112,17 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+## Integration Tests
+
+Our primary means of performing integration testing for libpod is with the
+[Ginkgo](https://github.com/onsi/ginkgo) BDD testing framework. This allows
+us to use native Golang to perform our tests and there is a strong affiliation
+between Ginkgo and the Go test framework.  Adequate test cases are expected to
+be provided with PRs.
+
+For details on how to run the tests for Podman in your test environment, see the
+Integration Tests [README](test).
```

Comment:
nit: I'd rather have this be `[README](test/README.md)`, although as long as we're rendered on GitHub it will be fine either way.  I'd also be fine with `... see the [test directory](test).`.

Reply:
I had one as README and another README.md before I pushed this and landed on README.  But After thinking about it, I think I prefer the suggested README.md.  Changed.